src/localization/en.js

```diff
@@ -58,4 +58,14 @@ export default {
     password: 'Password',
     button: 'Save changes',
   },
+
+  CREATE_TARGET: {
+    areaLenght: 'Specify area lenght',
```

Comment:
Typo in "length" here 😛

Reply:
I can't believe it. It's wrong in the figma design. I put it wrong everywhere 🙃